examples/models/llama2/export_llama_lib.py

```diff
@@ -57,6 +58,15 @@ def get_resource_path(resource_name) -> str:
     return pkg_resources.resource_filename(pkg_name, resource_name)
 
 
+def set_verbosity(val):
+    global verbosity_setting
+    verbosity_setting = val
+
+
+def verbose_export():
+    return verbosity_setting
```

Comment:
Would be good to clarify the type here. And along with it, `verbosity_setting: bool = False` above.

```suggestion
def set_verbosity(val: bool):
    global verbosity_setting
    verbosity_setting = val


def verbose_export() -> bool:
    return verbosity_setting
```